src/libs/Permissions.js

```diff
@@ -102,6 +102,10 @@ function canUseScanReceipts(betas) {
     return _.contains(betas, CONST.BETAS.SCAN_RECEIPTS) || canUseAllBetas(betas);
 }
 
+function canUseDistanceRequests(betas) {
```

Comment:
```suggestion
/**
 * @param {Array<String>} betas
 * @returns {Boolean}
 */
function canUseDistanceRequests(betas) {
```

Reply:
Oh yeah thanks I missed this one, my bad!